VimeoNetworking/Sources/Models/VIMVODItem.m

```diff
@@ -282,7 +282,7 @@ - (void)setVODItemType
 
 - (VIMVODConnection *)getVideosConnection
 {
-    return [self connectionWithName:VIMConnectionNameVideos];
+    return (VIMVODConnection *)[self connectionWithName:VIMConnectionNameVideos];
```

Comment:
@ghking Casting here to silence a warning about returning a VIMConnection instead of a VIMVODConnection.